<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BedMeshFrame</class>
 <widget class="QFrame" name="BedMeshFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QComboBox" name="profileComboBox">
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>Profile (default)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="bedMeshFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="addProfileButton">
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="removeProfileButton">
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QPushButton" name="recalibrateButton">
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Recalibrate</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
